src/archiefvernietigingscomponent/destruction/forms.py

```diff
@@ -52,10 +52,17 @@ class DestructionListForm(forms.ModelForm):
     reviewer_2 = forms.ModelChoiceField(
         queryset=User.objects.reviewers().all(), required=False
     )
+    contains_sensitive_info = forms.BooleanField(required=False)
```

Comment:
I don't think this line is necessary, since `contains_sensitive_info` is a model field

Reply:
Removed it!